### PR TITLE
Match LoadBattleDmaData

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

**Comment by JoshDuMan** [comment]
https://decomp.me/scratch/7Qfcv

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.00%) on `us`.

⚠️ This PR introduces 7 warnings:
- src/181810.c:216: warning: initialization from incompatible pointer type
- src/181810.c:223: warning: passing arg 1 of `dma_copy' makes pointer from integer without a cast
- src/181810.c:223: warning: passing arg 2 of `dma_copy' makes pointer from integer without a cast
- src/181810.c:223: warning: passing arg 3 of `dma_copy' makes pointer from integer without a cast
- src/181810.c:225: warning: passing arg 1 of `dma_copy' makes pointer from integer without a cast
- src/181810.c:225: warning: passing arg 2 of `dma_copy' makes pointer from integer without a cast
- src/181810.c:225: warning: passing arg 3 of `dma_copy' makes pointer from integer without a cast
- src/181810.c:483: warning: implicit declaration of function `start_rumble_type'
- src/181810.c:65: warning: implicit declaration of function `func_80263E08'
- src/181810.c:67: warning: implicit declaration of function `increment_status_menu_disabled'
- src/181810.c:88: warning: implicit declaration of function `decrement_status_menu_disabled'

File: include/common_structs.h [hunk 1]
```diff
@@ -92,6 +92,12 @@ typedef struct CamPosSettings {
     /* 0x10 */ Vec3f position;
 } CamPosSettings; // size = 0x1C
 
+typedef struct DmaTable {
+    /* 0x00 */ s32 dmaStart;
+    /* 0x04 */ s32 dmaEnd;
+    /* 0x08 */ s32 dmaDest;
+} DmaTable;
```
Comment (nanaian) [inline include/common_structs.h:99]: I think there's a nusys struct that looks like this

**Review by nanaian — APPROVED** [review]

File: include/common_structs.h (hunk 1 above)
Comment (ethteck) [reply include/common_structs.h:99]: If it's the one I'm thinking of, it's much larger than this

**Pull request merged**